src/client/index.js

```diff
@@ -386,6 +388,21 @@ function mkcircuit(data, opts) {
     show_scale();
     monitorview.on('change:start', show_range);
     monitorview.on('change:pixelsPerTick', show_scale);
+
+    let scaleIndex = 0;
+    $('button[name=zoom-in]').click(e => {
+        scaleIndex++;
+        paper.scale(Math.pow(1.1, scaleIndex));
+    });
+
+    $('button[name=zoom-out]').click(e => {
+        scaleIndex--;
+        paper.scale(Math.pow(1.1, scaleIndex));
+    });
+
+    paper.on('scale', (currentScale) => {
+       $('button[name=zoom-in]').prop('disabled', currentScale >= 5);
+    });
```

Comment:
This is essentially the same code as in https://github.com/tilk/digitaljs/pull/91. Could you factor it out?

Reply:
Honestly I don't know how the responsibility should be split. Could you recommend a solution? Which package should have this code implemented? Digitaljs or digialjs_online?

The idea that I have would be to implement it as a method of the `Circuit` class. Is it a good idea? What would be the alternatives?

Reply:
I think that your suggestion to add a method to the `Circuit` class is fine. Such a method should be able to add zooming capability to a given paper. It could set handlers on the buttons, or just return the handler code, so that the caller could apply it to appropriate elements, I don't know what's better.

Reply:
So I opened a new PR [(digitaljs#93)](https://github.com/tilk/digitaljs/pull/93) in digitaljs repository, and this should be merged before this PR is merged.

The code added in the last commit uses new methods implemented in the Circuit class in digitaljs package.